### PR TITLE
Add wiki command

### PR DESCRIPTION
**Pull request by Kas-tle**
Note: Algolia search API key was pushed intentionally; this is already publicly distributed as part of the wiki bundle.

## Usage
`/wiki <query>`

## Demo

https://github.com/user-attachments/assets/b07c1b16-c745-48c5-b234-49f4a22302ba

### CODE REVIEW

File: bot.properties.example [hunk 1]
```diff
@@ -12,3 +12,7 @@ github-token: github_oauth_token
 sentry-dsn: https://xxx@xxx.ingest.sentry.io/xxx
 sentry-env: production
 ocr-path: /usr/share/tesseract-ocr/4.00/tessdata
+algolia-application-id: 0DTHI9QFCH
+algolia-search-api-key: 3cc0567f76d2ed3ffdb4cc94f0ac9815
```
Comment (rtm516) [inline bot.properties.example:16]: Should we not use a separate API key if the portal lets us see search trends per key? Might be useful to see what people are looking for on discord vs the wiki its self.

Comment (Kas-tle) [reply bot.properties.example:16]: It doesn't let you do that, and it doesn't let you generate multiple API keys. You only get one public API key, and one write API key.

**Review by Chew — COMMENTED** [review]
This should be /wiki like it was originally, imo. Could do an alias !search for !wiki, /search should make /wiki show up since it's in the description anyway, but I'd need to test.

Everything else looks good, but I recommend 4 spaces for continuous indention so it looks better.

File: build.gradle [hunk 2]
```diff
@@ -89,6 +89,12 @@ dependencies {
     // Image processing and OCR
     implementation 'net.sourceforge.tess4j:tess4j:5.12.0'
     implementation 'org.imgscalr:imgscalr-lib:4.2'
+
+    // Agolia Search (For Wiki)
+    // We should eventually switch to v4, but currently it doesn't serialize POJOs correctly...
```
Comment (rtm516) [inline build.gradle:94]: Is this tracked somewhere on their issue tracker? If so can we get a link here

File: src/main/java/org/geysermc/discordbot/commands/search/SearchCommand.java [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+        getEmbedsFuture(query).whenComplete((embeds, throwable) -> {
+            if (throwable != null) {
+                MessageHelper.errorResponse(event, "Search Error",
+                        "An error occurred while searching for `" + query + "`");
+                return;
+            }
+
+            new PageHelper(embeds, event, -1);
+        });
+
```
Comment (rtm516) [inline src/main/java/org/geysermc/discordbot/commands/search/SearchCommand.java:116]:
```suggestion
```

**Review by rtm516 — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (Kas-tle) [reply build.gradle]: Figured out how to use v4.

**Comment by Kas-tle** [comment]
> This should be /wiki like it was originally, imo. Could do an alias !search for !wiki, /search should make /wiki show up since it's in the description anyway, but I'd need to test.
> 
> Everything else looks good, but I recommend 4 spaces for continuous indention so it looks better.

Changed to /wiki.

**Review by onebeastchris — APPROVED** [review]

File: src/main/java/org/geysermc/discordbot/listeners/SwearHandler.java [hunk 4]
```diff
@@ -200,4 +204,34 @@ public void onGuildMemberUpdateNickname(@NotNull GuildMemberUpdateNicknameEvent
...
+
+    //     ServerSettings.getLogChannel(event.getGuild()).sendMessageEmbeds(new EmbedBuilder()
+    //         .setTitle("Profanity removed")
+    //         .setDescription("**Sender:** " + event.getUser().getAsMention() + "\n" +
+    //                 "**Channel:** " + channel + "\n" +
+    //                 "**Regex:** `" + filterPattern + "`\n" +
+    //                 "**Command:** " + event.getCommandString())
+    //         .setColor(BotColors.FAILURE.getColor())
+    //         .build()).queue();
+    // }
```
Comment (rtm516) [inline src/main/java/org/geysermc/discordbot/listeners/SwearHandler.java:236]:
```suggestion
```

**Review by rtm516 — CHANGES_REQUESTED** [review]

**Review by rtm516 — APPROVED** [review]

**Pull request merged**